Misc/NEWS.d/next/Tests/2019-04-11-07-59-43.bpo-28009.s85urF.rst

```diff
@@ -0,0 +1,3 @@
+Fix the SkipUnless logic to be based on whether a program
```

Comment:
Slight rewording here: "Fix the uuid test skipping logic ..."

Reply:
fixed, but I'll push later.